android-project/app/build.gradle

```diff
@@ -24,7 +21,7 @@ android {
                 abiFilters 'arm64-v8a'
             }
             cmake {
-                arguments "-DANDROID_PLATFORM=android-19", "-DANDROID_STL=c++_static"
+                arguments "-DANDROID_PLATFORM=android-19", "-DANDROID_STL=c++_shared"
```

Comment:
C++ language features are used in a few c++ sources.
But SDL3 does not use the STL, so it can even be `none`.
We do that in `build-release.py`.
But when integrating SDL into bigger projects, it makes indeed sense to use `c++_shared`.

Reply:
"-DANDROID_STL=c++_static" prefab build not support static c++ runtime
"-DANDROID_STL=none"、"-DANDROID_STL=system" compile error, some code using std::xxx cause